app/commands/unlock_all_command.rb

```diff
@@ -0,0 +1,10 @@
+# UnlockAllCommand handles the `/deploy unlock all` command.
+class UnlockAllCommand < BaseCommand
+  def run
+    transaction do
+      active_locks = user.locks.active
```

Comment:
It feels weird that pull the active locks here, and then again in `unlock_all`. I think `unlock_all` should just return a list of the locks that were unlocked:

```ruby
active_locks = slashdeploy.unlock_all(user)
Slash.say UnlockedAllMessage.build active_locks
```